src/schema/user.js

```diff
@@ -18,7 +19,7 @@ export const typeDef = `
 type User {
   name: String!
   favoritePickUpBranch: String
-  bibliotekDkOrders: [BibliotekDkOrders!]!
+  bibliotekDkOrders(offset: Int limit: Int): BibliotekDkOrders!
```

Comment:
Istedet for Int på limit, så har vi en PaginationLimit type som kun tillader et nr. mellem 1 og 100 (Så man ikke kan spørge på eks. 5000)

Jeg kan også se at vi flere steder sætter offset og limit som required (!) -> vil det også give mening her?

Reply:
-yup. Har ændret det nu-

-Jeg har sat default værdi i selve userData service, så tænker det måske fint at det ikke er required. Så kan man lege med det på fbi-api uden at skulle forholde sig offset og limit.